src/components/AppHeader/AppHeader.vue

```diff
@@ -88,11 +89,15 @@ export default {
       type: Boolean,
       default: true,
     },
+    isSticky: {
+      type: Boolean,
+      default: false,
+    },
   },
 };
 </script>
 <style lang="scss" scoped>
-  .header {
+  .header > nav {
```

Comment:
I know we said we would try not to include other nav in the header, but just in case we do, I would make this `.header . navbar`

Reply:
Good point, but a .navbar is a bulma class that's added to the nav element in the header. So if we had another nav item we would also have another .navbar class, so it wouldn't make a difference, right?

Reply:
True. In that case, I'd recommend a specific class for this, so we can target exactly this `nav` element. Maybe `.main-nav`?

Reply:
That works.

Reply:
💯